source/stylesheets/modules/_year_filter.scss

```diff
@@ -5,11 +5,14 @@
 .year-filter {
 
   > .headline {
-    @include screenreaders-only;
+    @include filter-headline;
+    float: right;
```

Comment:
Floating seems a bit weird to me in this case since elements aren't placed next to each other.